integration/hive/src/main/java/org/apache/carbondata/hive/CarbonObjectInspector.java

```diff
@@ -60,14 +60,16 @@ public CarbonObjectInspector(final StructTypeInfo rowTypeInfo) {
   }
 
   private ObjectInspector getObjectInspector(final TypeInfo typeInfo) {
-    if (typeInfo.equals(TypeInfoFactory.doubleTypeInfo)) {
+    if (typeInfo.equals(TypeInfoFactory.stringTypeInfo)) {
```

Comment:
why this change and why move string up  ? assuming many columns will be string type ?

Reply:
yes, string should be compared first